src/util/clipboard.cpp

```diff
@@ -0,0 +1,35 @@
+#include "clipboard.h"
+
+#include <QApplication>
+#include <QClipboard>
+
+QString& Clipboard::text() {
+    static QString s_text;
+    return s_text;
+}
+
+void Clipboard::begin() {
+    text() = "";
+}
+
+void Clipboard::add(const QUrl& url) {
+    text().append(url.toString() + "\n");
+}
+
+void Clipboard::end() {
+    QApplication::clipboard()->setText(text());
```

Comment:
setText() only a handy wrapper for text. 
See https://github.com/qt/qtbase/blob/d6dacfecb1add4bfd3e4d0aeea5047dd95206588/src/gui/kernel/qclipboard.cpp#L297

Our data is a list of tracks instead. 

They can be copied to the systems Chipboard by 
QMimeData::setUrls()

This has been successfully implemented fro drag and drop here: 
https://github.com/mixxxdj/mixxx/blob/3de29630a80513127f8d3c0e5eb006922d240d08/src/util/dnd.cpp#L13

I think we should do the same for fully interoperability with other applications.

Reply:
This gives some insight for Linux:
https://www.medo64.com/2019/12/copy-to-clipboard-in-qt/
Not sure if this is still relevant.

Reply:
Hm, indeed it seems to make more sense to use QClipboard::setMimeData directly. Not sure why I didn't do that. I will give it a try. Thanks for the suggestion.

Reply:
Done